tests/integration/test_script_runner.py

```diff
@@ -655,6 +655,104 @@ class IntegrationTestFixture:
         ),
         extra_backend_dependencies=BackendDependencies.AWS,
     ),
+    IntegrationTestFixture(
```

Comment:
Maybe we should generate these test fixtures in a for loop?

Reply:
@anthonyburdi They are not all the same...

Reply:
Done.